yt_dlp/YoutubeDL.py

```diff
@@ -3675,10 +3664,7 @@ def run_pp(self, pp, infodict):
 
         if not files_to_delete:
             return infodict
-        if self.params.get('keepvideo', False):
-            for f in files_to_delete:
-                infodict['__files_to_move'].setdefault(f, '')
-        else:
+        if not self.params.get('keepvideo', False):
```

Comment:
Could I get eyes on this to confirm that this logic inversion works as expected now that I've deleted the `files_to_move` related stuff?